shmem.c

```diff
@@ -105,18 +105,18 @@ void newOverTimeClient() {
 }
 
 void addOverTimeClientSlot() {
-	// For each client slot, add an overTime slot
-	for(int i = 0; i < overTimeClientCount; i++) {
-		// Get the name of the shared memory
-		char *name = clientShmName(i);
+	// For each client slot, add pagesize overTime slots
+	for(int i = 0; i < overTimeClientCount; i++)
+	{
+		// Only increase size of the shm object is needed
+		if(shm_overTimeClients[i].size > (size_t)counters->overTime)
```

Comment:
Cast to `size_t` is done because `counters->overTime` is a signed integer whereas `shm_overTimeClients[i].size` is of type `size_t` (unsigned type)

Reply:
`shm_overTimeClients[i].size` stores the size of the memory in bytes whereas `counters->overTime` (effectively) stores the number of slots each overTime client should have, so the memory size should be divided by `sizeof(int)` to get the number of overTime slots stored in shared memory.